cmake/Reprojucer.cmake

```diff
@@ -1688,13 +1688,7 @@ function(jucer_export_target_configuration
   endif()
 
   if(DEFINED _EXTRA_LIBRARY_SEARCH_PATHS)
-    set(library_search_paths "")
-    foreach(path IN LISTS _EXTRA_LIBRARY_SEARCH_PATHS)
-      file(TO_CMAKE_PATH "${path}" path)
-      _FRUT_abs_path_based_on_jucer_project_dir(path "${path}")
-      list(APPEND library_search_paths "${path}")
-    endforeach()
-    set(JUCER_EXTRA_LIBRARY_SEARCH_PATHS_${config} "${library_search_paths}" PARENT_SCOPE)
+    set(JUCER_EXTRA_LIBRARY_SEARCH_PATHS_${config} "${_EXTRA_LIBRARY_SEARCH_PATHS}" PARENT_SCOPE)
```

Comment:
Now that `EXTRA_LIBRARY_SEARCH_PATHS` works like `HEADER_SEARCH_PATHS`, there should be a bit of code related to it below
https://github.com/McMartin/FRUT/blob/b874ee48365bfdfaaba029a1a452632e66434071/Jucer2CMake/src/reprojucer.hpp#L1410-L1413